deploy/operator.yaml

```diff
@@ -45,6 +45,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: DISABLE_TOPOLOGY_SPREAD_CONSTRAINTS
```

Comment:
I don't know about this. There are no other configurations that we use to set on the operator as env. Nobody will expect to find this option here.

Reply:
@guymguym another option would be adding a ConfigMap for env, but now noobaa operator does not have such a configuration we need to create one for that.

Reply:
I would prefer a label on the noobaa CR than env on the operator deployment.